### PR TITLE
Changes to OpenAPI schema descriptions

### PR DESCRIPTION
**Pull request by NathanD-GDS**
OpenAPI spec changes for James.

### CODE REVIEW

**Review by james-peacock-gds — CHANGES_REQUESTED** [review]

File: openapi/publicapi_spec.json [hunk 1]
```diff
@@ -1214,36 +1223,36 @@
...
+            "description" : "The amount, in pence, the user has paid or will pay. `amount` will match the value you sent in the request body.",
             "example" : 1200
           },
           "card_details" : {
             "$ref" : "#/components/schemas/CardDetails"
           },
           "created_date" : {
             "type" : "string",
-            "description" : "The date you created the payment.",
+            "description" : "The date and time you created this payment. This value uses Coordinated Universal Time (UTC) and ISO 8601 format - `YYYY-MM-DDTHH:MM:SSZ`.",
```
[the remark this answers is not in the document]
Comment (NathanD-GDS) [reply openapi/publicapi_spec.json:1234]: You're correct. I'll push a change to fix this now.

File: openapi/publicapi_spec.json [hunk 2]
```diff
@@ -1708,14 +1746,15 @@
           "payment_id" : {
             "type" : "string",
             "example" : "hu20sqlact5260q2nanm0q8u93",
-            "readOnly" : true
+            "readOnly" : true,
+            "description" : "The unique ID your payment service provider generated for this payment. This is not the same as the `payment_id`."
```
[the remark this answers is not in the document]
Comment (NathanD-GDS) [reply openapi/publicapi_spec.json:1750]: Christ, how have you spotted that?! You're right, fixing it now.

File: openapi/publicapi_spec.json [hunk 3]
```diff
@@ -1868,19 +1912,19 @@
         "properties" : {
           "capture_submit_time" : {
             "type" : "string",
-            "description" : "Date and time capture request has been submitted. May be null if capture request was not immediately acknowledged by payment gateway.",
+            "description" : "The date and time GOV.UK Pay asked your payment service provider to take the payment from your user’s account. This value uses Coordinated Universal Time (UTC) and ISO 8601 format - `YYYY-MM-DDTHH:MM:SSZ`.",
```
[the remark this answers is not in the document]
Comment (NathanD-GDS) [reply openapi/publicapi_spec.json:1915]: Fixing now

File: openapi/publicapi_spec.json [hunk 4]
```diff
@@ -1973,17 +2019,20 @@
...
-            "readOnly" : true
+            "readOnly" : true,
+            "description" : "The amount refunded to the user in pence."
           },
           "created_date" : {
             "type" : "string",
             "example" : "2017-01-10T16:52:07.855Z",
-            "readOnly" : true
+            "readOnly" : true,
+            "description" : "The date and time you created this refund.<br><br>This value uses Coordinated Universal Time (UTC) and ISO 8601 format - `YYYY-MM-DDTHH:MM:SSZ`."
```
[the remark this answers is not in the document]
Comment (NathanD-GDS) [reply openapi/publicapi_spec.json:2029]: Fixing now

**Pull request closed without merging**